src/simplejavatexteditor/SupportedKeywords.java

```diff
@@ -31,7 +31,7 @@
                 "union", "while", "asm", "dynamic_cast", "namespace", "reinterpret_cast", "try",
                 "bool", "explicit", "new", "static_cast", "typeid", "catch", "false", "operator",
                 "template", "typename", "class", "friend", "private", "this", "using", "const_cast",
-                "inline", "public", "throw", "virtual", "delete", "mutable", "protected", "true", "wchar_t" };
+                "inline", "public", "throw", "virtual", "delete", "mutable", "protected", "true", "wchar_t , int_t , uint32_t , and , goto , char8_t , char16_t , char32_t" };
```

Comment:
No quotes?

Reply:
Sorry, apparently I missed it, I'm not very strong in java, I initially started learning C ++, I stumbled upon your project, saw an array with keywords and added it

Reply:
Can you fix this then @sp1cklerBoy? :)